assets/section-footer.css

```diff
@@ -1,11 +1,8 @@
-.footer {
+.footer-wrapper--border-top,
```

Comment:
Small nitpick:
Maybe the class could be something like `footer--separator-lines` ?

I think the current name sound like it's applying border only to the wrapper and at the top when it's doing it in a couple places. Might be more self explanatory this way. 

Also if we try and follow [BEM conventions](https://github.com/Shopify/themes/wiki/BEM-convention) for class naming, we shouldn't be using `footer-wrapper--border-top` as we don't have a `footer-wrapper` class on this element 👍

Reply:
Yeah, I wasn't sure about this. I borrowed the nomenclature from the header separator implementation, which uses `header-wrapper--border-bottom`. If we want to rename to `separator-lines`, I think we should update that to match. 

Either way, good call on the `footer-wrapper` name though... I've updated this classname to `footer--border-top` in https://github.com/Shopify/dawn/pull/1776/commits/67f0744b6f2fd71101e96e8aef6339b0dc8b517c.